deployment/network-operator/values.yaml

```diff
@@ -93,7 +93,7 @@ sriov-network-operator:
     operator: nvcr.io/nvidia/mellanox/sriov-network-operator:network-operator-1.4.0
     sriovConfigDaemon: nvcr.io/nvidia/mellanox/sriov-network-operator-config-daemon:network-operator-1.4.0
     sriovCni: ghcr.io/k8snetworkplumbingwg/sriov-cni:v2.6.3
-    ibSriovCni: ghcr.io/k8snetworkplumbingwg/ib-sriov-cni:848e3b4c97c17d8cb0fcf09aa1358edd0354db4f
+    ibSriovCni: ghcr.io/k8snetworkplumbingwg/ib-sriov-cni:v1.0.2
```

Comment:
shouldnt we set to v1.0.3 ?

Reply:
I'll bump all dependencies in a separate PR